src/ui/pages/SwapForm/Quotes/RateLine.tsx

```diff
@@ -91,79 +130,143 @@ export function RateLine({
   const protocolBorderRadius = shouldCircleProtocolImages ? '50%' : undefined;
 
   return (
-    <HStack
-      gap={8}
-      justifyContent="space-between"
-      style={{
-        visibility:
-          !isLoading && !selectedQuote && !error ? 'hidden' : undefined,
-      }}
-    >
-      <UIText kind="small/regular" color="var(--neutral-700)">
-        Rate
-      </UIText>
-      <span>
-        {isLoading && !selectedQuote ? (
-          <span style={{ color: 'var(--neutral-500)' }}>
-            Fetching offers...
-          </span>
-        ) : selectedQuote ? (
-          <HStack
-            // in design it's 4, but design has circle images
-            gap={gap}
-            style={{
-              // Prevent formatted rate text from changing width
-              // when the values change. This way, the animated images stay in one place
-              fontVariantNumeric: 'tabular-nums',
-            }}
-          >
-            {selectedQuote.contract_metadata?.icon_url ? (
-              <div
+    <>
```

Comment:
Is this needed?

Reply:
Hm. There are couple of dialog in the end of this file on the root level. So I wrapped the whole JSX in <></>